### DISCUSSION TITLE
Creality E3V2 with sonic pad. Acceleration

### DISCUSSION THREAD

**Discussion by girenloland** [OrcaSlicer/OrcaSlicer#1442] [General]
Machine: Ender 3 V2 with sonic pad.

When I make my custom profile, I do it the same way I do in PrusaSlicer.
First I choose Ender 3 V2 printer,  edit my start G code, then set the machine limits. 

Then I set my speeds and accelerations before slicing and printing.
When I set acceleration to 5000 inner walls. 3000 outer walls. Infill 6500. Speed at 100mm\s
Orca sets acceleration to 1000 across the board. This is set in gcode when first layer starts. Not changing anything after that in G code. G code flavor is set to Klipper.
I have set machine limit to 6500 X and Y.  

Is there anything I am doing wrong and will make me feel stupid here or?
I checked the .json files for ender 3 v2. Max default acceleration is set to 5000

**Comment by santiagomoneta** [comment]
any updates on this one? not using a Sonic pad but have my Ender 3 v2 on Klipper and trying to find the best way to optimize the default ender3v2 printer profile...

**Reply by santiagomoneta** [reply]
Maybe on the MOTION ABILITY tab, uncheck the option to emit the limits in the code and let the firmware to take care of that
